internal/kibana/synthetics/create.go

```diff
@@ -26,7 +26,12 @@ func (r *Resource) Create(ctx context.Context, request resource.CreateRequest, r
 		return
 	}
 
-	namespace := plan.SpaceID.ValueString()
+	// Use namespace if explicitly set, otherwise fall back to space_id
+	namespace := plan.Namespace.ValueString()
+	if namespace == "" || plan.Namespace.IsNull() || plan.Namespace.IsUnknown() {
+		namespace = plan.SpaceID.ValueString()
+	}
+
```

Comment:
```suggestion
```

The monitor should always be created in the space defined by `space_id`, can you revert this change.